From 10e6ca1d517def513cad0272d60a1dbef7577143 Mon Sep 17 00:00:00 2001
From: Tom Gundersen <teg@jklm.no>
Date: Tue, 20 Nov 2018 17:09:44 +0100
Subject: [PATCH] units/system: add `messagebus` alias

This makes sure we are pulled in by services that depend on the old
sysvinit name for dbus.

Signed-off-by: Tom Gundersen <teg@jklm.no>
---
 src/units/system/dbus-broker.service.in | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/units/system/dbus-broker.service.in b/src/units/system/dbus-broker.service.in
index 1ccf4db..f3bae16 100644
--- a/src/units/system/dbus-broker.service.in
+++ b/src/units/system/dbus-broker.service.in
@@ -15,4 +15,6 @@ ExecStart=@bindir@/dbus-broker-launch --scope system --audit
 ExecReload=@bindir@/busctl call org.freedesktop.DBus /org/freedesktop/DBus org.freedesktop.DBus ReloadConfig
 
 [Install]
-Alias=dbus.service
+# Make sure that services can still refer to this under the name of the
+# old SysV script (messagebus).
+Alias=dbus.service messagebus.service
-- 
2.19.1

